datastore/adapter/keytransform.py

```diff
@@ -163,6 +187,7 @@ class _NamespaceAdapter(_Adapter[DS], typing.Generic[DS]):
 		>>> ds.get(datastore.Key('/a/b/c/d'))
 		'cd'
 	"""
+	__slots__ = ()
```

Comment:
We should add `namespace` in `__slots__`

Reply:
Doing that here (or in several other `_<something>` places) unfortunately causes Python [to raise an error as it cannot establish a stable class field order](https://stackoverflow.com/q/53060607/277882). As such, I've opted to put put these names only into the public outer classes (`KeyTransform{Object,Binary}Adapter` in this case) to make things work correctly.

Reply:
Ok